book/installation/binaries.md

```diff
@@ -26,3 +26,15 @@ As an example, you could install the Linux x86_64 version like so:
 1. Test the binary with `./reth --version` (it should print the version).
 2. (Optional) Move the `reth` binary to a location in your `PATH`, so the `reth` command can be called from anywhere.  
    For most Linux distros, you can move the binary to `/usr/local/bin`: `sudo cp ./reth /usr/local/bin`.
+
+Alternatively, you can use [binstall](https://github.com/cargo-bins/cargo-binstall) to install and load the binary:
+
+1. To install `binstall`, run: 
+   ```bash
+   curl -L --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/cargo-bins/cargo-binstall/main/install-from-binstall-release.sh | bash
+   ```
+2. Then, install `reth` with:
+   ```bash
+   cargo binstall reth
+   ```
```

Comment:
Please put this under a "package managers" section or something similar so we can expand it later, and just mention that we support `cargo binstall` (with a link to it). Explaining how to install it here imo doesn't make sense

Reply:
`binstall` is not a package manager though, it's only for installing crates which have corresponding binary releases in their designated repository, we can remove the installation bit if that would sounds good (open to more fitting terminology, not sure what that would be)

Reply:
Ok just removing the installation steps is fine, we can figure out how to format it later when we add other channels

Reply:
can also just copy:

https://nexte.st/book/pre-built-binaries.html?highlight=binstall#using-cargo-binstall

Reply:
Done 👍